Extensions/EdFi.Ods.Profiles.Sample/EdFi.Ods.Profiles.Sample.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This is weird that these changes are showing up here, they should have gone in with #9.  Please rebase this branch so we can see what if any changes are being done to the profile project files.